lib/client.js

```diff
@@ -12,20 +12,21 @@
 
 	/**
 	 * Performs a NodeJS require followed by a fallback function call on failure.
+	 * If you need a certain property only, you can specify moduleProp.
 	 * @param module The name of a NodeJS module
-	 * @param [fallback] Optional fallback function that must return a substitute.
+	 * @param [moduleProp] The name of the specific property of module.
+	 * @param [fallback] fallback function that must return a substitute.
 	 */
-	var safeRequire = function (module, fallback) {
+	var safeRequire = function (module, moduleProp, fallback) {
```

Comment:
It would be nice if moduleProp is not a required argument so we can call the function with just the module and fallback function.